lib/resque/plugins/status.rb

```diff
@@ -231,24 +231,23 @@ def failed(*messages)
       def completed(*messages)
         set_status({
           'status' => STATUS_COMPLETED,
-          'message' => "Completed at #{Time.now}"
+          'message' => nil,
```

Comment:
Better '' or just drop the change. We definitely don't want to leave an intermediate message after complete.